### ISSUE TITLE
Minify dist stylesheets

### ISSUE THREAD

**Issue by JayPanoz** [readium/readium-css#90]
I'm submitting a feature request

**Short description of the issue/suggestion:**

We’ve kept dist stylesheets DEV-ish (comments, etc.) so far but we should probably minify dist stylesheets given they may be injected in every EPUB file, etc. If we output map files, dev needs should be covered.

Moreover, minifying CSS is quite a strong recommandation for web perf. 

See [postcss-clean](https://www.npmjs.com/package/postcss-clean)

Re. #89

**Comment by JayPanoz** [comment]
Thanks to Daniel for recap’ing our discussion during the Readium Call. New version is available in the `develop` branch, with highlights and references to source maps removed. In addition, minification of dist stylesheets has been put in place. 

So this will be automatically closed once develop is merged into master.
